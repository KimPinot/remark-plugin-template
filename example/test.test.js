import fs from "fs";
import path from "path";
import { remark } from "remark";
import remarkEmbed from "../src/index.js";

const doc = fs.readFileSync(path.resolve(process.cwd(), "example/example.md"));

const result = (markdown) => remark().use(remarkEmbed).processSync(markdown);

describe("remark-template", () => {
  test("match doc snapshot", () => {
    expect(result(doc).value).toMatchSnapshot();
  });
});
