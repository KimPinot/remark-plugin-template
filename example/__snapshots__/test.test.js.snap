// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`remark-template match doc snapshot 1`] = `
"# h1

this is test node
"
`;
